docs/medical-api/api-reference/patient/bulk-create-patient.mdx

```diff
@@ -0,0 +1,250 @@
+    The ID number. For type `ssn`, should be 9 digits.
+  </ParamField>
+  
+  <ParamField body="driversLicenceState" type="string">
+    The 2 letter state acronym, for example: `CA`. 
+    Required if `driversLicenceNo` is provided.
+  </ParamField>
+</Expandable>
+
+**Example**
+
+The first Patient below has two email addresses, the second Patient only has one email, and the
+third Patient has three email addresses:
+
+| ... | zip\* | email-1          | email-2             | email-3           | ... |
+| --- | ----- | ---------------- | ------------------- | ----------------- | --- |
+| ... | 12345 | some@domain.com  | another@domain.com  |                   | ... |
+| ... | 12312 | avocado@food.com |                     |                   | ... |
+| ... | 99887 | lawyer@job.com   | personal@domain.com | family@domain.com | ... |
+
+For address fields, they work as a group: if you provide `addressLine1-2`, then all address fields
+with index `2` will be required (otherwise, the respective row/Patient will be marked as invalid).
+Following this logic, if you don't provide any address field for index `3` through `10`, all the
+address fields for those indexes are disconsidered, and the Patient is considered valid (assuming
+other non-address fields are valid too).
+
+At least one complete address must be provided:
+
+- addressLine1-1
+- addressLine2-1
+- city-1
+- state-1
+- zip-1
+
+For information about the result file, see [this](/medical-api/handling-data/webhooks#result-csv-file).
+
+### Validation
+
+This endpoint validates access to the provided Facility ID (or that there's only one if omitted).
+It will return a `404` if the facility can't be found or `400` if there's more than one Facility
+and the ID was not provided.
+
+Once the CSV is uploaded, the Metriport API will also validate the file is in the expected format
+(CSV). It migh set the status of the job to `failed` and send a corresponding webhook message.
+
+If those previous validations are successful, each row will be validated individually and it's
+respective status will be stored so it can be reviewed in the
+[result CSV file](/medical-api/handling-data/webhooks#result-csv-file) (it won't fail the whole
+job at this point).
+
+### Uploading the CSV
+
+The URL is pre-signed to your account and has a duration of 10 minutes. It should only be used
+once. Use it to send a `PUT` request like so:
+
+```shell
+curl --location --request PUT '<UPLOAD-URL>' \
+--header 'Content-Type: text/csv' \
+--header 'x-api-key: <YOUR-API-KEY>' \
+--data-binary '@/local-to-your/file.csv'
+```
+
+#### Webhook with the result
+
+Once the CSV file is uploaded, the Metriport API will start processing it and will send
+[webhook messages](/medical-api/handling-data/webhooks#bulk-patient-create) to your App
+indicating the status of the processing, including the resulting CSV file containing the
+newly created Metriport IDs for the successfully created Patients.
```

Comment:
So to understand correctly were sending a webhook when we start processing the csv but its after we've createed the patients? Given the metriportIds will be on the csv we return?

Reply:
Yeah, I'll rewrite to indicate to different moments/events.

Reply:
done